.github/workflows/main-ecr.yml

```diff
@@ -0,0 +1,198 @@
+name: Build and Push Heracles docker image to ECR
```

Comment:
this is atlas image.. we should update the description to `Build and Push Atlas docker image to ECR`